python/taichi/lang/__init__.py

```diff
@@ -16,6 +16,7 @@
 import taichi.lang.linalg_impl
 import taichi.lang.meta
 from taichi._lib import core as _ti_core
+print(_ti_core.__file__)
```

Comment:
not necessary?

Reply:
This was added for resolving an installation issue, it is now removed.